ansible/setup.yml

```diff
@@ -0,0 +1,102 @@
+- name: Install MLPerf Storage v2.0
+  hosts: clients
+  become: true
+  tasks:
+    - name: COMMON | Ensure curl, git and python build dependencies are installed
```

Comment:
please keep names consistent... for example add `COMMON | ` to all common steps and same for RHEL, SLES, UBUNTU...